sealog-auxDataUDPgrabber.py

```diff
@@ -161,17 +164,17 @@ def handle_ctm_packet(packet):
         }
     )
 
-
 def handle_icl_packet(packet):
```

Comment:
Since the packet is slightly different, I would prefer a separate `handle_iclc_packet` function rather than overloading one function to handle both. They look sufficiently different, are generated in different places, etc.

Reply:
Ok, I can see sense in that. I'll add a separate "ICLC" parser as well. I should have that pushed in a few minutes.